pytorch/edgeml_pytorch/trainer/fastTrainer.py

```diff
@@ -50,43 +58,41 @@ def __init__(self, FastObj, numClasses, sW=1.0, sU=1.0,
         self.numMatrices = self.FastObj.num_weight_matrices
         self.totalMatrices = self.numMatrices[0] + self.numMatrices[1]
 
-        self.optimizer = self.optimizer()
-
         self.RNN = BaseRNN(self.FastObj, batch_first=self.batch_first).to(self.device)
-
-        self.FC = nn.Parameter(torch.randn(
-            [self.FastObj.output_size, self.numClasses])).to(self.device)
-        self.FCbias = nn.Parameter(torch.randn(
-            [self.numClasses])).to(self.device)
-
+        self.simpleFC = SimpleFC(self.FastObj.output_size, self.numClasses).to(self.device)
```

Comment:
a better name for the instance instead of self.simpleFC (so that we are generic enough to fit the classifier() function.